From b32fdde901c10cda223648c125654fbf577b4886 Mon Sep 17 00:00:00 2001
From: "wei, wushuangx" <wushuangx.wei@intel.com>
Date: Wed, 23 Feb 2022 23:17:40 +0800
Subject: [PATCH] use private libdrm instead of external libdrm

Rename lib names to libxxx_orig, so that we use
libdrm-intel instead of external libdrm, and
bunches of libdrm_pri renaming patches could
be removed.

Tracked-On: OAM-88717
Signed-off-by: yifang ma <yifangx.ma@intel.com>
Signed-off-by: Chenglei Ren <chenglei.ren@intel.com>
Signed-off-by: wei, wushuangx <wushuangx.wei@intel.com>
---
 Android.bp                        | 14 +++++++-------
 Android.sources.bp                |  2 +-
 amdgpu/Android.bp                 | 10 +++++-----
 amdgpu/Android.sources.bp         |  2 +-
 data/Android.bp                   |  4 ++--
 etnaviv/Android.bp                | 10 +++++-----
 etnaviv/Android.sources.bp        |  2 +-
 freedreno/Android.bp              | 10 +++++-----
 freedreno/Android.sources.bp      |  2 +-
 intel/Android.bp                  | 10 +++++-----
 intel/Android.sources.bp          |  2 +-
 libkms/Android.bp                 | 18 +++++++++---------
 libkms/Android.sources.bp         | 10 +++++-----
 nouveau/Android.bp                | 10 +++++-----
 nouveau/Android.sources.bp        |  2 +-
 omap/Android.bp                   |  6 +++---
 omap/Android.sources.bp           |  2 +-
 radeon/Android.bp                 |  6 +++---
 radeon/Android.sources.bp         |  2 +-
 tegra/Android.bp                  |  6 +++---
 tests/Android.bp                  |  4 ++--
 tests/modetest/Android.bp         | 12 ++++++------
 tests/modetest/Android.sources.bp |  2 +-
 tests/proptest/Android.bp         |  8 ++++----
 tests/util/Android.bp             | 14 +++++++-------
 tests/util/Android.sources.bp     |  2 +-
 26 files changed, 86 insertions(+), 86 deletions(-)

diff --git a/Android.bp b/Android.bp
index dbac5626..cbd98379 100644
--- a/Android.bp
+++ b/Android.bp
@@ -22,7 +22,7 @@
 //
 
 package {
-    default_applicable_licenses: ["external_libdrm_license"],
+    default_applicable_licenses: ["external_libdrm_license_orig"],
 }
 
 // Added automatically by a large-scale-change that took the approach of
@@ -40,7 +40,7 @@ package {
 // used in the current project.
 // See: http://go/android-license-faq
 license {
-    name: "external_libdrm_license",
+    name: "external_libdrm_license_orig",
     visibility: [":__subpackages__"],
     license_kinds: [
         "SPDX-license-identifier-BSD",
@@ -54,7 +54,7 @@ subdirs = ["*"]
 build = ["Android.sources.bp"]
 
 cc_defaults {
-    name: "libdrm_defaults",
+    name: "libdrm_defaults_orig",
     cflags: [
         // XXX: Consider moving these to config.h analogous to autoconf.
         "-DMAJOR_IN_SYSMACROS=1",
@@ -77,7 +77,7 @@ cc_defaults {
 }
 
 cc_library_headers {
-    name: "libdrm_headers",
+    name: "libdrm_headers_orig",
     vendor_available: true,
     host_supported: true,
     defaults: ["libdrm_defaults"],
@@ -86,13 +86,13 @@ cc_library_headers {
 
 // Library for the device
 cc_library {
-    name: "libdrm",
+    name: "libdrm_orig",
     recovery_available: true,
     vendor_available: true,
     host_supported: true,
     defaults: [
-        "libdrm_defaults",
-        "libdrm_sources",
+        "libdrm_defaults_orig",
+        "libdrm_sources_orig",
     ],
 
     export_include_dirs: ["include/drm", "android"],
diff --git a/Android.sources.bp b/Android.sources.bp
index 73356dd8..1e4f8013 100644
--- a/Android.sources.bp
+++ b/Android.sources.bp
@@ -1,7 +1,7 @@
 // Autogenerated with Android.sources.bp.mk
 
 cc_defaults {
-    name: "libdrm_sources",
+    name: "libdrm_sources_orig",
     srcs: [
         "xf86drm.c",
         "xf86drmHash.c",
diff --git a/amdgpu/Android.bp b/amdgpu/Android.bp
index 94f5668d..370a0abe 100644
--- a/amdgpu/Android.bp
+++ b/amdgpu/Android.bp
@@ -4,22 +4,22 @@ package {
     // all of the 'license_kinds' from "external_libdrm_license"
     // to get the below license kinds:
     //   SPDX-license-identifier-MIT
-    default_applicable_licenses: ["external_libdrm_license"],
+    default_applicable_licenses: ["external_libdrm_license_orig"],
 }
 
 build = ["Android.sources.bp"]
 
 cc_library_shared {
-    name: "libdrm_amdgpu",
+    name: "libdrm_amdgpu_orig",
 
     cflags: [
 	"-DAMDGPU_ASIC_ID_TABLE=\"/vendor/etc/hwdata/amdgpu.ids\""
     ],
 
     defaults: [
-        "libdrm_defaults",
-        "libdrm_amdgpu_sources",
+        "libdrm_defaults_orig",
+        "libdrm_amdgpu_sources_orig",
     ],
     vendor: true,
-    shared_libs: ["libdrm"],
+    shared_libs: ["libdrm_orig"],
 }
diff --git a/amdgpu/Android.sources.bp b/amdgpu/Android.sources.bp
index be85283d..653ef89b 100644
--- a/amdgpu/Android.sources.bp
+++ b/amdgpu/Android.sources.bp
@@ -1,7 +1,7 @@
 // Autogenerated with Android.sources.bp.mk
 
 cc_defaults {
-    name: "libdrm_amdgpu_sources",
+    name: "libdrm_amdgpu_sources_orig",
     srcs: [
 	"amdgpu_asic_id.c",
         "amdgpu_bo.c",
diff --git a/data/Android.bp b/data/Android.bp
index 8bb59211..6faf6066 100644
--- a/data/Android.bp
+++ b/data/Android.bp
@@ -4,11 +4,11 @@ package {
     // all of the 'license_kinds' from "external_libdrm_license"
     // to get the below license kinds:
     //   SPDX-license-identifier-MIT
-    default_applicable_licenses: ["external_libdrm_license"],
+    default_applicable_licenses: ["external_libdrm_license_orig"],
 }
 
 prebuilt_etc {
-    name: "amdgpu.ids",
+    name: "amdgpu.ids_orig",
     proprietary: true,
     sub_dir: "hwdata",
     src: "amdgpu.ids",
diff --git a/etnaviv/Android.bp b/etnaviv/Android.bp
index d8d04af1..1c5ebdef 100644
--- a/etnaviv/Android.bp
+++ b/etnaviv/Android.bp
@@ -4,17 +4,17 @@ package {
     // all of the 'license_kinds' from "external_libdrm_license"
     // to get the below license kinds:
     //   SPDX-license-identifier-MIT
-    default_applicable_licenses: ["external_libdrm_license"],
+    default_applicable_licenses: ["external_libdrm_license_orig"],
 }
 
 build = ["Android.sources.bp"]
 
 cc_library_shared {
-    name: "libdrm_etnaviv",
+    name: "libdrm_etnaviv_orig",
     defaults: [
-        "libdrm_defaults",
-        "libdrm_etnaviv_sources",
+        "libdrm_defaults_orig",
+        "libdrm_etnaviv_sources_orig",
     ],
     vendor: true,
-    shared_libs: ["libdrm"],
+    shared_libs: ["libdrm_orig"],
 }
diff --git a/etnaviv/Android.sources.bp b/etnaviv/Android.sources.bp
index aa828900..aaed1558 100644
--- a/etnaviv/Android.sources.bp
+++ b/etnaviv/Android.sources.bp
@@ -1,7 +1,7 @@
 // Autogenerated with Android.sources.bp.mk
 
 cc_defaults {
-    name: "libdrm_etnaviv_sources",
+    name: "libdrm_etnaviv_sources_orig",
     srcs: [
         "etnaviv_device.c",
         "etnaviv_gpu.c",
diff --git a/freedreno/Android.bp b/freedreno/Android.bp
index ca7f3e28..9aeefa6d 100644
--- a/freedreno/Android.bp
+++ b/freedreno/Android.bp
@@ -4,17 +4,17 @@ package {
     // all of the 'license_kinds' from "external_libdrm_license"
     // to get the below license kinds:
     //   SPDX-license-identifier-MIT
-    default_applicable_licenses: ["external_libdrm_license"],
+    default_applicable_licenses: ["external_libdrm_license_orig"],
 }
 
 build = ["Android.sources.bp"]
 
 cc_library_shared {
-    name: "libdrm_freedreno",
+    name: "libdrm_freedreno_orig",
     defaults: [
-        "libdrm_defaults",
-        "libdrm_freedreno_sources",
+        "libdrm_defaults_orig",
+        "libdrm_freedreno_sources_orig",
     ],
     vendor: true,
-    shared_libs: ["libdrm"],
+    shared_libs: ["libdrm_orig"],
 }
diff --git a/freedreno/Android.sources.bp b/freedreno/Android.sources.bp
index 3c1ca316..808608c5 100644
--- a/freedreno/Android.sources.bp
+++ b/freedreno/Android.sources.bp
@@ -1,7 +1,7 @@
 // Autogenerated with Android.sources.bp.mk
 
 cc_defaults {
-    name: "libdrm_freedreno_sources",
+    name: "libdrm_freedreno_sources_orig",
     srcs: [
         "freedreno_device.c",
         "freedreno_pipe.c",
diff --git a/intel/Android.bp b/intel/Android.bp
index 7c82f1e6..58b7a55b 100644
--- a/intel/Android.bp
+++ b/intel/Android.bp
@@ -28,19 +28,19 @@ package {
     // to get the below license kinds:
     //   SPDX-license-identifier-BSD
     //   SPDX-license-identifier-MIT
-    default_applicable_licenses: ["external_libdrm_license"],
+    default_applicable_licenses: ["external_libdrm_license_orig"],
 }
 
 build = ["Android.sources.bp"]
 
 cc_library_shared {
-    name: "libdrm_intel",
+    name: "libdrm_intel_orig",
     defaults: [
-        "libdrm_defaults",
-        "libdrm_intel_sources",
+        "libdrm_defaults_orig",
+        "libdrm_intel_sources_orig",
     ],
     vendor: true,
 
     // Removed dependency to libpciaccess: not used on Android
-    shared_libs: ["libdrm"],
+    shared_libs: ["libdrm_orig"],
 }
diff --git a/intel/Android.sources.bp b/intel/Android.sources.bp
index 46e0328f..e968e2e7 100644
--- a/intel/Android.sources.bp
+++ b/intel/Android.sources.bp
@@ -1,7 +1,7 @@
 // Autogenerated with Android.sources.bp.mk
 
 cc_defaults {
-    name: "libdrm_intel_sources",
+    name: "libdrm_intel_sources_orig",
     srcs: [
         "intel_bufmgr.c",
         "intel_bufmgr_fake.c",
diff --git a/libkms/Android.bp b/libkms/Android.bp
index d46bab4f..d7042b3a 100644
--- a/libkms/Android.bp
+++ b/libkms/Android.bp
@@ -4,21 +4,21 @@ package {
     // all of the 'license_kinds' from "external_libdrm_license"
     // to get the below license kinds:
     //   SPDX-license-identifier-MIT
-    default_applicable_licenses: ["external_libdrm_license"],
+    default_applicable_licenses: ["external_libdrm_license_orig"],
 }
 
 build = ["Android.sources.bp"]
 
 cc_library_shared {
-    name: "libkms",
+    name: "libkms_orig",
     defaults: [
-        "libdrm_defaults",
-        "libkms_sources",
-        "libkms_vmwgfx_sources",
-        "libkms_intel_sources",
-        "libkms_nouveau_sources",
-        "libkms_radeon_sources",
+        "libdrm_defaults_orig",
+        "libkms_sources_orig",
+        "libkms_vmwgfx_sources_orig",
+        "libkms_intel_sources_orig",
+        "libkms_nouveau_sources_orig",
+        "libkms_radeon_sources_orig",
     ],
     vendor: true,
-    shared_libs: ["libdrm"],
+    shared_libs: ["libdrm_orig"],
 }
diff --git a/libkms/Android.sources.bp b/libkms/Android.sources.bp
index 5582f235..3addc57d 100644
--- a/libkms/Android.sources.bp
+++ b/libkms/Android.sources.bp
@@ -1,7 +1,7 @@
 // Autogenerated with Android.sources.bp.mk
 
 cc_defaults {
-    name: "libkms_sources",
+    name: "libkms_sources_orig",
     srcs: [
         "linux.c",
         "dumb.c",
@@ -10,28 +10,28 @@ cc_defaults {
 }
 
 cc_defaults {
-    name: "libkms_vmwgfx_sources",
+    name: "libkms_vmwgfx_sources_orig",
     srcs: [
         "vmwgfx.c",
     ],
 }
 
 cc_defaults {
-    name: "libkms_intel_sources",
+    name: "libkms_intel_sources_orig",
     srcs: [
         "intel.c",
     ],
 }
 
 cc_defaults {
-    name: "libkms_nouveau_sources",
+    name: "libkms_nouveau_sources_orig",
     srcs: [
         "nouveau.c",
     ],
 }
 
 cc_defaults {
-    name: "libkms_radeon_sources",
+    name: "libkms_radeon_sources_orig",
     srcs: [
         "radeon.c",
     ],
diff --git a/nouveau/Android.bp b/nouveau/Android.bp
index b844fdde..84d7ae4a 100644
--- a/nouveau/Android.bp
+++ b/nouveau/Android.bp
@@ -4,17 +4,17 @@ package {
     // all of the 'license_kinds' from "external_libdrm_license"
     // to get the below license kinds:
     //   SPDX-license-identifier-MIT
-    default_applicable_licenses: ["external_libdrm_license"],
+    default_applicable_licenses: ["external_libdrm_license_orig"],
 }
 
 build = ["Android.sources.bp"]
 
 cc_library_shared {
-    name: "libdrm_nouveau",
+    name: "libdrm_nouveau_orig",
     defaults: [
-        "libdrm_defaults",
-        "libdrm_nouveau_sources",
+        "libdrm_defaults_orig",
+        "libdrm_nouveau_sources_orig",
     ],
     vendor: true,
-    shared_libs: ["libdrm"],
+    shared_libs: ["libdrm_orig"],
 }
diff --git a/nouveau/Android.sources.bp b/nouveau/Android.sources.bp
index 5ecdb53c..72012e55 100644
--- a/nouveau/Android.sources.bp
+++ b/nouveau/Android.sources.bp
@@ -1,7 +1,7 @@
 // Autogenerated with Android.sources.bp.mk
 
 cc_defaults {
-    name: "libdrm_nouveau_sources",
+    name: "libdrm_nouveau_sources_orig",
     srcs: [
         "nouveau.c",
         "pushbuf.c",
diff --git a/omap/Android.bp b/omap/Android.bp
index a3a5a9e3..ccabfbd6 100644
--- a/omap/Android.bp
+++ b/omap/Android.bp
@@ -4,18 +4,18 @@ package {
     // all of the 'license_kinds' from "external_libdrm_license"
     // to get the below license kinds:
     //   SPDX-license-identifier-MIT
-    default_applicable_licenses: ["external_libdrm_license"],
+    default_applicable_licenses: ["external_libdrm_license_orig"],
 }
 
 build = ["Android.sources.bp"]
 
 cc_library_shared {
-    name: "libdrm_omap",
+    name: "libdrm_omap_orig",
     defaults: [
         "libdrm_defaults",
         "libdrm_omap_sources",
     ],
     vendor: true,
 
-    shared_libs: ["libdrm"],
+    shared_libs: ["libdrm_orig"],
 }
diff --git a/omap/Android.sources.bp b/omap/Android.sources.bp
index 3c7da94a..2491fa9a 100644
--- a/omap/Android.sources.bp
+++ b/omap/Android.sources.bp
@@ -1,7 +1,7 @@
 // Autogenerated with Android.sources.bp.mk
 
 cc_defaults {
-    name: "libdrm_omap_sources",
+    name: "libdrm_omap_sources_orig",
     srcs: [
 	"omap_drm.c",
     ],
diff --git a/radeon/Android.bp b/radeon/Android.bp
index 6c52084f..540f3947 100644
--- a/radeon/Android.bp
+++ b/radeon/Android.bp
@@ -4,17 +4,17 @@ package {
     // all of the 'license_kinds' from "external_libdrm_license"
     // to get the below license kinds:
     //   SPDX-license-identifier-MIT
-    default_applicable_licenses: ["external_libdrm_license"],
+    default_applicable_licenses: ["external_libdrm_license_orig"],
 }
 
 build = ["Android.sources.bp"]
 
 cc_library_shared {
-    name: "libdrm_radeon",
+    name: "libdrm_radeon_orig",
     defaults: [
         "libdrm_defaults",
         "libdrm_radeon_sources",
     ],
     vendor: true,
-    shared_libs: ["libdrm"],
+    shared_libs: ["libdrm_orig"],
 }
diff --git a/radeon/Android.sources.bp b/radeon/Android.sources.bp
index 820ac4d6..3d004f02 100644
--- a/radeon/Android.sources.bp
+++ b/radeon/Android.sources.bp
@@ -1,7 +1,7 @@
 // Autogenerated with Android.sources.bp.mk
 
 cc_defaults {
-    name: "libdrm_radeon_sources",
+    name: "libdrm_radeon_sources_orig",
     srcs: [
         "radeon_bo_gem.c",
         "radeon_cs_gem.c",
diff --git a/tegra/Android.bp b/tegra/Android.bp
index 297c0193..6bb9dde2 100644
--- a/tegra/Android.bp
+++ b/tegra/Android.bp
@@ -4,13 +4,13 @@ package {
     // all of the 'license_kinds' from "external_libdrm_license"
     // to get the below license kinds:
     //   SPDX-license-identifier-MIT
-    default_applicable_licenses: ["external_libdrm_license"],
+    default_applicable_licenses: ["external_libdrm_license_orig"],
 }
 
 cc_library_shared {
-    name: "libdrm_tegra",
+    name: "libdrm_tegra_orig",
     vendor: true,
-    shared_libs: ["libdrm"],
+    shared_libs: ["libdrm_orig"],
 
     srcs: ["tegra.c"],
 
diff --git a/tests/Android.bp b/tests/Android.bp
index bff7f991..666d2347 100644
--- a/tests/Android.bp
+++ b/tests/Android.bp
@@ -4,12 +4,12 @@ package {
     // all of the 'license_kinds' from "external_libdrm_license"
     // to get the below license kinds:
     //   SPDX-license-identifier-MIT
-    default_applicable_licenses: ["external_libdrm_license"],
+    default_applicable_licenses: ["external_libdrm_license_orig"],
 }
 
 subdirs = ["*"]
 
 cc_library_headers {
-    name: "libdrm_test_headers",
+    name: "libdrm_test_headers_orig",
     export_include_dirs: ["."],
 }
diff --git a/tests/modetest/Android.bp b/tests/modetest/Android.bp
index 02a17fe9..172f8799 100644
--- a/tests/modetest/Android.bp
+++ b/tests/modetest/Android.bp
@@ -4,18 +4,18 @@ package {
     // all of the 'license_kinds' from "external_libdrm_license"
     // to get the below license kinds:
     //   SPDX-license-identifier-MIT
-    default_applicable_licenses: ["external_libdrm_license"],
+    default_applicable_licenses: ["external_libdrm_license_orig"],
 }
 
 build = ["Android.sources.bp"]
 
 cc_test {
-    name: "modetest",
+    name: "modetest_orig",
     defaults: [
-        "libdrm_defaults",
-        "modetest_sources",
+        "libdrm_defaults_orig",
+        "modetest_sources_orig",
     ],
 
-    shared_libs: ["libdrm"],
-    static_libs: ["libdrm_util"],
+    shared_libs: ["libdrm_orig"],
+    static_libs: ["libdrm_util_orig"],
 }
diff --git a/tests/modetest/Android.sources.bp b/tests/modetest/Android.sources.bp
index c6aca2ef..e510bfe8 100644
--- a/tests/modetest/Android.sources.bp
+++ b/tests/modetest/Android.sources.bp
@@ -1,7 +1,7 @@
 // Autogenerated with Android.sources.bp.mk
 
 cc_defaults {
-    name: "modetest_sources",
+    name: "modetest_sources_orig",
     srcs: [
         "buffers.c",
         "cursor.c",
diff --git a/tests/proptest/Android.bp b/tests/proptest/Android.bp
index e13d060b..61aacfdf 100644
--- a/tests/proptest/Android.bp
+++ b/tests/proptest/Android.bp
@@ -4,14 +4,14 @@ package {
     // all of the 'license_kinds' from "external_libdrm_license"
     // to get the below license kinds:
     //   SPDX-license-identifier-MIT
-    default_applicable_licenses: ["external_libdrm_license"],
+    default_applicable_licenses: ["external_libdrm_license_orig"],
 }
 
 cc_test {
-    name: "proptest",
+    name: "proptest_orig",
     defaults: ["libdrm_defaults"],
     srcs: ["proptest.c"],
 
-    shared_libs: ["libdrm"],
-    static_libs: ["libdrm_util"],
+    shared_libs: ["libdrm_orig"],
+    static_libs: ["libdrm_util_orig"],
 }
diff --git a/tests/util/Android.bp b/tests/util/Android.bp
index 0322c2ac..4f6f3d94 100644
--- a/tests/util/Android.bp
+++ b/tests/util/Android.bp
@@ -27,18 +27,18 @@ package {
     // all of the 'license_kinds' from "external_libdrm_license"
     // to get the below license kinds:
     //   SPDX-license-identifier-MIT
-    default_applicable_licenses: ["external_libdrm_license"],
+    default_applicable_licenses: ["external_libdrm_license_orig"],
 }
 
 build = ["Android.sources.bp"]
 
 cc_library_static {
-    name: "libdrm_util",
+    name: "libdrm_util_orig",
     defaults: [
-        "libdrm_defaults",
-        "libdrm_util_sources",
+        "libdrm_defaults_orig",
+        "libdrm_util_sources_orig",
     ],
-    shared_libs: ["libdrm"],
-    header_libs: ["libdrm_test_headers"],
-    export_header_lib_headers: ["libdrm_test_headers"],
+    shared_libs: ["libdrm_orig"],
+    header_libs: ["libdrm_test_headers_orig"],
+    export_header_lib_headers: ["libdrm_test_headers_orig"],
 }
diff --git a/tests/util/Android.sources.bp b/tests/util/Android.sources.bp
index 529e1124..c9a6f1a0 100644
--- a/tests/util/Android.sources.bp
+++ b/tests/util/Android.sources.bp
@@ -1,7 +1,7 @@
 // Autogenerated with Android.sources.bp.mk
 
 cc_defaults {
-    name: "libdrm_util_sources",
+    name: "libdrm_util_sources_orig",
     srcs: [
         "format.c",
         "kms.c",
-- 
2.34.1

